source/troubleshooting.rst

```diff
@@ -38,11 +38,11 @@ Cleaning out the Database
 -------------------------
 Our command to run a postgres container does implement docker volumes. Volumes allow our postgres data to be persistent, which means that the data will remain even if the postgres container is removed and recreated. If for any reason you would like to delete your postgres database, you will need to remove the docker volume.
 
-Removing the volume for RHEL 6 or Centos 6
+Removing the volume for RHEL 6 and Centos 6
 
 # rm -rf /var/lib/docker/volumes/qpc-data
 
-Removing the volume for RHEL 7, Centos 7, Fedora 27, or Fedora 28
+Removing the volume for RHEL 7 and Centos 7
```

Comment:
RHEL 7 or Centos 7
(same grouped things comment)
**ALSO** Do a global search of the entire document for CentOS. I just double-checked the https://www.centos.org/ website. CentOS (not Centos) should be everywhere.

Reply:
Done.